.travis.yml

```diff
@@ -1,20 +1,26 @@
 sudo: false
 language: go
-go:
-  - "1.10"
-  - "1.11.x"
-  - tip
+matrix:
+  include:
+  - go: '1.10'
+  - go: 1.11.x
+  - go: tip
+  - go: tip
+    env: SNOWFLAKE_AZURE=true
 before_install:
-  - openssl aes-256-cbc -k "$super_secret_password" -in parameters.json.enc -out parameters.json -d
-  - openssl aes-256-cbc -k "$super_secret_password" -in rsa-2048-private-key.p8.enc -out rsa-2048-private-key.p8 -d
-  - curl -L -s https://github.com/golang/dep/releases/download/v0.3.2/dep-linux-amd64 -o $GOPATH/bin/dep
-  - chmod +x $GOPATH/bin/dep
+- ./parameter_setup.sh
+- openssl aes-256-cbc -k "$super_secret_password" -in rsa-2048-private-key.p8.enc
+  -out rsa-2048-private-key.p8 -d
+- curl -L -s https://github.com/golang/dep/releases/download/v0.3.2/dep-linux-amd64
+  -o $GOPATH/bin/dep
+- chmod +x $GOPATH/bin/dep
 install:
-  - dep ensure
+- dep ensure
```

Comment:
In my impression, there is not much difference between `install` and `before_install` phase as long as the tests run  regularly. Merge before_install and install into one install phase in a script.